packages/nx-plugin/tsconfig.json

```diff
@@ -3,6 +3,11 @@
   "compilerOptions": {
     "baseUrl": "src"
   },
-  "include": ["./**/*.ts"],
+  "include": [
+    "./**/*.ts",
+    "schemas/cdk-service/files/contracts/healthContract.ts__tmpl__",
+    "schemas/cdk-service/files/functions/health/config.ts__tmpl__",
+    "schemas/cdk-service/files/vitest.setup.ts__tmpl__"
```

Comment:
👀 
is this working because files don't have variables in their templating ?

Reply:
i think my IDE added those automatically while editing, nice catch 🎣